net-libs/libbtbb/files/libbtbb-2020.12.1-musl-u-char.patch

```diff
@@ -0,0 +1,13 @@
+# Fix unknown type name 'u_char' on musl
+# Closes: https://bugs.gentoo.org/829245
+# Closes: https://bugs.gentoo.org/715758
```

Comment:
I would suggest to submit this patch to upstream and link the PR here (or upstream commit if it will be quickly accepted).

Reply:
@arkamar noted. I'll do it

Reply:
Could you reference https://github.com/greatscottgadgets/libbtbb/pull/64 here in the patch? This is useful refernece for devs, because they can check the state of that PR with next version bump, for example.